### PR TITLE
Testing multi-container applications with Docker Compose and Semaphore

### PR DESCRIPTION
**Pull request by ervinb**

### CODE REVIEW

File: source/docs/docker/testing-multicontainer-apps-with-docker-compose-and-semaphore.md.erb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+`docker-compose.yml`, the Docker-enabled platform will be pre-selected.
+
+<img src="" alt="Choosing a platform" class="img-responsive img-bordered">
+
+On the next screen, you can optionally configure your container registry
+credentials for accessing your private Docker images.
+
+<img src="" alt="Setting up container registry credentials" class="img-responsive img-bordered">
+
+Next, Semaphore will suggest some build commands. If you need to, you can tailer
```
Comment (nikolalsvk) [inline source/docs/docker/testing-multicontainer-apps-with-docker-compose-and-semaphore.md.erb]: `tailer` => `tailor`

File: source/docs/docker/testing-multicontainer-apps-with-docker-compose-and-semaphore.md.erb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+<img src="" alt="Setting up build commands" class="img-responsive img-bordered">
+
+Clicking "Build" will kick-off your first build. Sit back, and let Semaphore do
+all the work. After a short while, you will be greeted with a successful build.
+
+<img src="" alt="First successful build with Docker Compose and Semaphore" class="img-responsive img-bordered">
+
+Testing images is great, but at some point, you will want to push them to your
+container registry. To see how it's done, check out [our guides about deploying
+images to the clouds](nd-cr-deployments).
```
Comment (nikolalsvk) [inline source/docs/docker/testing-multicontainer-apps-with-docker-compose-and-semaphore.md.erb]: maybe `to the clouds` => `into the cloud`

File: source/docs/docker.md.erb [hunk 3]
```diff
@@ -15,3 +15,7 @@ in your build commands.
 
 To avoid permission problems when using the Docker CLI, commands should be used
 with `sudo`.
+
+Guides:
+
+- [Testing multi-container applications with Compose and Semaphore](/docs/docker/testing-multicontainer-apps-with-docker-compose-and-semaphore.html)
```
Comment (markoa) [inline source/docs/docker.md.erb]: you mean docker-compose

**Comment by ervinb** [comment]
@dunjardl The PR is ready for copy-editing

**Pull request merged**